source/codegen/metadata/nirfmxinstr_restricted/functions.py

```diff
@@ -1,86 +1,46 @@
 functions = {
-    'RegisterSpecialClientSnapshotInterest': {
```

Comment:
Most of the changes in this file are non-functional. I've alphabetized the functions. I have a comment below identifying the one functional change.